pravega/pravega-sensor-collector
# Garbage completed files delete schedule task changes

**Change log description**
In last check-in we added feature to delete processing completed files. In this check-in we are making schedule to delete completed files after a particular time interval.
**Purpose of the change**
Garbage collect processing completed files at particular scheduled interval.
**What the code does**
It create scheduled job, which after particular interval (default 12 hrs i.e. 720 mins) calls FileProcessor's deleteCompletedFiles to fetch completed files list from DB and delete one after another from DB and stored directory location.
**How to verify it**
Change property "PRAVEGA_SENSOR_COLLECTOR_RAW1_DELETE_COMPLETED_FILES_INTERVAL_IN_MINUTES" value to 1 (i.e. 1 min.) , after that start PSC and observe Completed_Files directory(created parallel to DB file), after files processing files will get moved to Completed_Files directory and after a minute all the files in Completed_Files directory get deleted.
**What tests you have run. (Manually, automated whatever). How can someone else test this?**
Follow steps mentioned above.
